harrier/extensions.py

```diff
@@ -103,9 +96,9 @@ def load(self):
                     self._extensions[ext_type].extend([(path_match, attr) for path_match in attr.path_matches])
                 elif ext_type:
                     self._extensions[ext_type].append(attr)
-                elif any(getattr(attr, n, False) is True for n in filter_attrs):
+                elif getattr(attr, filter_attr, None):
                     self._extensions[ExtType.template_filters][attr_name] = attr
-                elif any(getattr(attr, n, False) is True for n in function_attrs):
+                elif getattr(attr, function_attr, None) or hasattr(attr, 'jinja_pass_arg'):
```

Comment:
surely `jinja_pass_arg` should be added to `function_attrs` and this doesn't need changing?

Are there tests for context functions etc.?

Reply:
Why bother? Jinja 3.1 won't support the old methods so seems irrelevant.

Reply:
well it definitely doesn't make sense to have `function_attr` as a variable defined elsewhere and `'jinja_pass_arg'` as a string here, I don't mind which way around, but it should be consistent.

Reply:
Sure